src/engine/objects/robot/sensors/SimColorSensor.ts

```diff
@@ -79,7 +80,7 @@ export class SimColorSensor extends SimComplexSensor {
       density: 1,
       isSensor: true,
       userData,
-      filterCategoryBits: EntityCategory.SENSORS,
+      filterCategoryBits: EntityCategory.ROBOT_PART,
       filterMaskBits: EntityMask.SENSORS,
```

Comment:
I think we should limit the color sensor to interact only with zones.
We should not be able to have the robot on top of wall or object but no point in fight with strange behavior that might occur because of this.
We can use `filterMaskBits:: EntityCategory.ZONES` as it is the only bit we need.